tests/merlin_hashtable_test.cc.cu

```diff
@@ -1944,6 +1944,367 @@ void test_evict_strategy_customized_correct_rate(size_t max_hbm_for_vectors,
 
   CudaCheckError();
 }
+
+void test_insert_or_assign_multi_threads(
+      size_t max_hbm_for_vectors, 
+      bool use_constant_memory, 
```

Comment:
The constant memory configuration has been removed, please rebase on master and process the conflicts, thanks.